src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1,11 +1,15 @@
 databaseChangeLog:
   - include:
-      file: db/changelog/changes/01-create-books-table.yaml
+      file: db/changelog/changes/01-create-books-table.yml
```

Comment:
it's totally okay to use .yaml, even more preferable. Windows used to have file extensions of three letters, now it's no longer strict